confidential-ledger-demo/server/confidentialledger.py

```diff
@@ -1,224 +1,90 @@
-# Copyright (c) 2021 Microsoft.
-# Copyright (c) 2021 Avanade Inc.
+# Import the Azure authentication library
```

Comment:
We do need Avanade & MSFT inc copyrights

Reply:
added